backend-tests/tests/test_create_org.py

```diff
@@ -67,10 +67,21 @@ def test_success(self, clean_migrated_mongo):
         smtp_mock.assert_called()
         logging.info("TestCreateOrganizationEnterprise: Assert ok.")
 
-        r = uc.call(
-            "POST", useradm.URL_LOGIN, auth=("some.user@example.com", "asdfqwer1234")
-        )
-        assert r.status_code == 200
+        # Try log in every second for 3 minutes.
+        # - There usually is a slight delay (in order of ms) for propagating
+        #   the created user to the db.
+        for i in range(3*60):
+            rsp = api.call(
+                "POST",
+                useradm.URL_LOGIN,
+                auth=("some.user@example.com", "asdfqwer1234")
+            )
+            if rsp.status_code == 200:
+                break
+            time.sleep(1)
```

Comment:
If this is a common pattern, we might think about wrapping it in some prebaked function.

Reply:
we are moving tests from integration to backend integration so there is quite a lot happening in the backend integration tests recently;
we're trying to be careful when moving things to a common part of the test, but next time I'll need this pattern I'll move it there (to testutils/common)